skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -76,6 +79,18 @@ abstract class AbstractSkaffoldRunConfiguration(
      * Persisted Skaffold config file absolute path for Skaffold run configurations.
      * See more at [com.intellij.openapi.vfs.VirtualFile.getPath]
      */
-    @Attribute("skaffoldConfigurationFilePath")
+    @get:OptionTag("skaffoldConfigurationFilePath")
     var skaffoldConfigurationFilePath: String? = null
+
+    override fun readExternal(element: Element) {
+        super.readExternal(element)
+
+        XmlSerializer.deserializeInto(this, element)
+    }
+
+    override fun writeExternal(element: Element) {
+        super.writeExternal(element)
+
+        XmlSerializer.serializeInto(this, element)
```

Comment:
Can you use a `PersistantStateComponent` instead?

Reply:
We could, but it would essentially do the same thing, and there is a dangerous mix of `loadState()` methods from `RunConfigurationBase` and `PersistantStateComponent`. The only difference is that we would probably create the DOM Element ourselves.

Reply:
ok. lgtm.